internal/pipeline/resolver/repository.go

```diff
@@ -71,7 +71,7 @@ func filterPipelines(repoURLs []string, org string, client *buildkite.Client) ([
 
 func getRepoURLs(r *git.Repository) ([]string, error) {
 	if r == nil {
-		return nil, fmt.Errorf("Could not determine current repository")
+		return nil, fmt.Errorf("could not determine current repository")
```

Comment:
@jradtilbrook , go style decisions on [error strings](https://google.github.io/styleguide/go/decisions.html#error-strings )

Reply:
Awesome! I always forget which way to go. Could you check with golangci lint if there is a rule we can add so this can fail at the linting stage and then we can all get consistent 🙌. Create a card for it to add in